### PR TITLE
Remove warnings, deprecation from 1.23 manual

### PR DESCRIPTION
**Pull request by upadhyeammit**
Removed deprecation and warnings from 1.23 manual.

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@tbrisker request to check. Thank You !

**Comment by tbrisker** [comment]
@upadhyeammit sorry, i may have not explained myself clearly, these should be removed from nightly manual, not 1.23.

**Comment by upadhyeammit** [comment]
@tbrisker pushed the changes, request to check. Thank You !

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -6,15 +6,4 @@ This section will be updated prior to the next release.
 
 ### Release Notes
 
-### Deprecations
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: Please keep the section headers

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -6,15 +6,4 @@ This section will be updated prior to the next release.
...
-
-* Ubuntu Xenial (16.04) is deprecated and will no longer be supported starting from Foreman 1.25. Users are urged to upgrade their base OS to Ubuntu Bionic (18.04).
-* [#27048](https://projects.theforeman.org/issues/27048) Smart Proxy DNS module methods `get_name, get_name!, get_ipv4_address, get_ipv4_address!, get_ipv6_address, get_ipv6_address!, get_resource_as_string, get_resource_as_string!` were marked as deprecated and scheduled for removal in 1.24. They have been unused in the Smart Proxy since 1.16 but kept around for plugins. If you're a Smart Proxy DNS Plugin author, make sure you migrate away from them before 1.24.
-
-### Upgrade warnings
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]: this one too

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Pull request merged**